<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>morphology</class>
 <widget class="QWidget" name="morphology">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" rowspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="selectButton">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dilateButton">
         <property name="text">
          <string>Dilate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="erosionButton">
         <property name="text">
          <string>Erosion</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openButton">
         <property name="text">
          <string>Opening</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colseButton">
         <property name="text">
          <string>Closing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Kernel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="kernelBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>280</height>
        </size>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>70</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>40</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QScrollArea" name="oriArea">
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>575</width>
        <height>396</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QScrollArea" name="openArea">
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>575</width>
        <height>395</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QScrollArea" name="erosionArea">
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>575</width>
        <height>395</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
